docs/LSPSupport.md

```diff
@@ -135,6 +135,31 @@ This menu action either opens the reference in a popup or navigates to the refer
 
 [textDocument/references](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_references) is used also via [Find Usages](./UserGuide.md#find-usages) to show references.
 
+#### External References

```

Comment:
Added documentation for this opt-in feature including how/when/why it can be enabled.